test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -1,4 +1,4 @@
-define(['jobCommChannel', 'base/js/namespace', 'common/runtime', 'testUtil'], (
```

Comment:
As before, you don't need to set `common/jobCommChannel` as long as the alias in `narrative_paths.js` is updated.